src/crustls.h

```diff
@@ -392,4 +392,13 @@ rustls_result rustls_server_session_write_tls(rustls_server_session *session,
                                               size_t count,
                                               size_t *out_n);
 
+/**
+ * Write up to `len` characters of the SNI hostname into `buf`. If the
+ * handshake has not been performed yet or if the client does not support SNI,
+ * the returned length in out_n will be 0.
+ * https://docs.rs/rustls/0.19.0/rustls/struct.ServerSession.html#method.get_sni_hostname
+ */
+rustls_result rustls_server_session_sni_hostname_get(rustls_server_session *session,
```

Comment:
nit: `rustls_server_session_get_sni_hostname` seems like a more natural way to translate the `rustls` method.